### ISSUE TITLE
Test

### ISSUE THREAD

**Issue by Rushor** [Rushor/Forge-of-Souls#1]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)

**Current behavior**: Tell us what happens in detail (Where, When, What, Why, Who)

**Expected behavior**: Tell us what should happen instead

**Steps to reproduce the problem**:

1. Test
2. 
3. 

**Include proofs for this behavior**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures, pets, minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or http://wotlk.cavernoftime.com/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**
